src/DocumentationBrowserViewExtension/DocumentationBrowserViewExtension.cs

```diff
@@ -70,7 +70,7 @@ public override void Loaded(ViewLoadedParams viewLoadedParams)
             this.documentationBrowserMenuItem = new MenuItem { Header = Resources.MenuItemText, IsCheckable = true };
             this.documentationBrowserMenuItem.Checked += MenuItemCheckHandler;
             this.documentationBrowserMenuItem.Unchecked += MenuItemUnCheckedHandler;
-            this.viewLoadedParamsReference.AddMenuItem(MenuBarType.View, this.documentationBrowserMenuItem);
+            this.viewLoadedParamsReference.AddExtensionMenuItem(this.documentationBrowserMenuItem);
```

Comment:
Thanks, I think this makes most sense given none of the other MenuBarTypes will be supported for the first version of this API, if we plan to support them in the future, we can add another version of this API.